### DISCUSSION TITLE
Is there any guide that would describe what links are needed for ComfyUI? And also for Stable Diffusion WebUI Forge ?

### DISCUSSION THREAD

**Discussion by andrej-ka** [LykosAI/StabilityMatrix#998] [General]
Hello,
Windows 10, RTX4070 TI Super, i9-13900K, 64gb RAM

I'm trying to run Stability Matrix + ComfyUI + Flux
Now I found that D:\AI\Data\Packages\ComfyUI\models\Unet 
does not have a link to d:\AI\Data\Models\Unet

Is there any guide that would describe what links are needed for ComfyUI? And also for Stable Diffusion WebUI Forge ?

Because with Stable Diffusion WebUI Forge I also have problems with Flux
I get AssertionError: You do not have CLIP state dict if I connect flux_dev SafeTensor - full - fp32 - 22.17 GB

But if I connect flux_dev SafeTensor - pruned - fp8 - 15.91 GB
Then it works

Could you help with this? Any ideas would be helpful
Thanks

**Comment by mohnjiles** [comment]
Comfy has changed the name of the `unet` folder to `diffusion_models`, which is where we link our shared Unet folder to. 

Your issues with Flux are due to most Flux models needing a separate text encoder & vae. Please check the examples for instructions: https://comfyanonymous.github.io/ComfyUI_examples/flux/
